docs/web/examples/rootscript.py

```diff
@@ -19,21 +19,21 @@
 from twisted.web import vhost, static, script, server
 from twisted.application import internet, service
 
-default = static.Data('text/html', '')
+default = static.Data("text/html", "")
```

Comment:
Shouldn't the second string here be `bytes`?

Reply:
The second parameter to twisted.web.static.Data is documented as `str`: https://github.com/twisted/twisted/blob/trunk/src/twisted/web/static.py#L57

At some point it will be good to get type annotations for more of twisted.web into the tree, such as this PR: https://github.com/twisted/twisted/pull/1284/

Reply:
Fixed by changing this to:

```
default = static.Data(b"", "text/html")
```